Requesting for ashikmuhammed.is-a.dev subdomain.

AshikCEH opened the pull request:
<!--
!!!
YOU MUST FILL OUT THIS TEMPLATE ENTIRELY FOR YOUR PR TO BE ACCEPTED, IT IS NOT OPTIONAL.
!!!
-->

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->

<!-- Change each checkbox to [x] to mark it as checked. Do not keep the spaces between the parentheses. -->

- [x] I have **read** and **agree** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x] I understand my domain will be removed if I violate the [Terms of Service](https://is-a.dev/terms).
- [x] My file is in the `domains` directory and has the `.json` file extension.
- [x] My file's name is lowercased and alphanumeric. <!-- Your file's name is yourname.json, not YourName.json or your_name.json. -->
- [x] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied websites. -->
- [x] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g., X, Discord) for contact. -->
- [ ] I have provided sufficient reasoning for obtaining NS records. <!-- You can ignore this if you are not applying for NS records. -->

# Website Preview
<!-- Provide a link or screenshot of your website below. You MUST complete this step for your PR to be approved. -->

<!-- Uncomment the following line if you are applying for NS records, and provide your reasoning here. -->
<!-- # NS Records Reasoning -->


## Netlify Site for Preview.
`ashikmuhammed.netlify.app`

Commit:
Create AshikMuhammed.json

Commit:
Merge branch 'is-a-dev:main' into main

roxiTheReal commented:
1. You didn't check any of the checkboxes
2. The maintainers already know what the subdomain name should be based off the JSON file's name
3. The JSON is in the wrong format, check the [documentation](https://docs.is-a.dev)
4. Uncapitalize the file name
5. You don't need to include the GitHub username in the PR, but you must do so in the JSON file

iostpa commented on domains/AshikMuhammed.json:
All letters should be lowercase.

iostpa commented on domains/AshikMuhammed.json:
```diff
@@ -0,0 +1,10 @@
+        "username": "AshikCEH",
+        "email": "jinnsulthan@proton.me"
+    },
+    "record": {
```
```suggestion
    "records": {
```

iostpa reviewed (changes requested):
I would recommend to add more stuff to your website like a quick about me.
(Also I don't think you have to check all of the boxes but it might differ from each maintainer)

Commit:
Update and rename AshikMuhammed.json to ashikmuhammed.json

Commit:
Merge branch 'main' into main

AshikCEH commented:
**Apologies for the earlier issues — I've now fixed everything as requested:**

1.  I've reviewed and checked all the required boxes, agreeing to the terms and conditions.
2.  The JSON file has been properly updated and formatted as per the guidelines.
3.  Renamed the JSON file to lowercase, as recommended.
4.  I've also pushed my latest commits, including updates to the website and preview based on my latest code.

I know my site might still feel a bit like a "hello world" demo 😅 — I’m still learning and just getting started with web development. Thanks for your patience and understanding!

roxiTheReal commented:
For a Netlify website, you have to go through a different process. Please refer to the [docs](https://docs.is-a.dev).

AshikCEH commented:
**"Yes, I have gone through the provided documentation. I have also added the custom domain on Netlify."**

AshikCEH commented:
> So, are you saying that I need to update my code to the following?

```json
{
  "owner": {
    "username": "AshikCEH",
    "email": "example@gmail.com"
  },
  "record": {
    "A": ["75.2.60.5"]
  }
}
```

roxiTheReal commented:
Yes, but with `"record"` changed to `"records"`.

Commit:
Update ashikmuhammed.json

AshikCEH commented:
> Okay, thank you! The JSON code above is just a sample — that’s why I used a placeholder email address as an example.

> I’ve also updated the changes as you mentioned on Json.

roxiTheReal commented:
Perfect, thanks. Now everything should be good to go. Just wait until a maintainer merges the changes.

AshikCEH commented:
**Thank you, bro! You really helped clear up my doubts and confusion. I appreciate it 😊**

roxiTheReal commented:
No need to. And next time, don't use an AI for stuff like this. They're bound to fail a lot.

Commit:
Merge branch 'main' into main

professorSergioMarquina commented:
> I would recommend to add more stuff to your website like a quick about me. (Also I don't think you have to check all of the boxes but it might differ from each maintainer)

**Dude I have completed the pendings as you said.**

iostpa reviewed (dismissed):
I just saw this PR now in our github logs :/

iostpa committed:
Update ashikmuhammed.json

iostpa reviewed (approved):
It was totally complete fr fr
Anyways, it should be good now
